tests/queries/0_stateless/00626_in_syntax.sql

```diff
@@ -28,16 +28,10 @@ select 1 in (0 + 1, 1, toInt8(sin(5)));
 select (0 + 1, 1, toInt8(sin(5))) in (0 + 1, 1, toInt8(sin(5)));
 select identity(tuple(1)) in (tuple(1), tuple(2));
 select identity(tuple(1)) in (tuple(0), tuple(2));
-select identity(tuple(1)) in (identity(tuple(1)), tuple(2));
-select identity(tuple(1)) in (identity(tuple(0)), tuple(2));
-select identity(tuple(1)) in (identity(tuple(1)), identity(tuple(2)));
-select identity(tuple(1)) in (identity(tuple(1)), identity(identity(tuple(2))));
-select identity(tuple(1)) in (identity(tuple(0)), identity(identity(tuple(2))));
```

Comment:
I'm not sure about this. Without this change the test used to fail because `identity()` [forbids](https://github.com/ClickHouse/ClickHouse/blob/532b341de921178c6621db7f10093a9cce760641/src/Functions/identity.cpp#L21) constant folding. And the documentation says about the operator IN:

```
The right side of the operator can be a set of constant expressions, a set of tuples with constant expressions (shown in the examples above), or the name of a database table or SELECT subquery in brackets.
```